code_generation/enum.h

```diff
@@ -84,6 +84,14 @@ class KODE_EXPORT Enum
      */
     void setIsQENUM(bool qenum = true);
 
+    /**
+     * @brief setTypedef
+     * This method can be used to generate typedef for the given enum.
+     * Using typefeds is not recommended in C++ code, mostly makes sense using it generating C code.
```

Comment:
Typo: typefed -> typedef

"makes sense using it" is lacking a subject. Suggested rephrase: `Using typedefs is mostly useful for C code and not recommended in C++ code.`